examples/multitenancy/hello-world/hs1.yaml

```diff
@@ -3,6 +3,5 @@ kind: HelloWorldService
 metadata:
   name: hs1
 spec:
-  greeting: Hello hello hello
-  replicas: 1 
```

Comment:
I checked hello-world-chart-0.0.3.tgz. It actually contains replicas field. So let's not remove it. The error that you were seeing was probably because an earlier/manual installation of HelloWorldService backed by 0.0.2 hello-world-chart. That version does not have replicas field in values.yaml.

Reply:
Ok sounds good, I will add it back in.